### ISSUE TITLE
Improve overall Accessibility 

### ISSUE THREAD

**Issue by whizkydee** [whizkydee/dropd#5]
Right now, a couple of things are missing in terms of accessibility of the elements. These include:

* Proper tab indexing
* Functional focus and blur states... etc

We should fix this.
